app/assets/javascripts/_page_show.es6

```diff
@@ -113,12 +113,14 @@ const COMPONENT_CLASSES = [
             const fileUrl = element.href;
             const fileId = element.getAttribute('data-resource-id');
 
-            fetchSignedResource(
-                filePath,
-                fileUrl,
-                `a[data-resource-id="${fileId}"]`,
-                'file'
-            );
+            fetchSignedResource(filePath, fileUrl)
+                .then(signedUrl => {
+                        debugger
+                        const fileElement = document.querySelector(`a[data-resource-id="${fileId}"]`);
+                        if (fileElement) {
+                            fileElement.href = signedUrl;
+                        }
```

Comment:
I went to make sure this is working with a console.log while testing locally and while it is definitely replacing the url I strongly suspect it had a signed url already in place which would mean this was redundant to begin with, will need to dig into it a bit more to verify...

Reply:
This seems to be working fine for me. I uploaded some file attachments in DEV and will monitor that they still work after the signed URL expiration window. I think it's fine to remove the debugger now 👍

Reply:
doh I didnt realize I left a debugger, jeeze louise